python/pyspark/sql/connect/client/retries.py

```diff
@@ -248,13 +249,6 @@ def __iter__(self) -> Generator[AttemptManager, None, None]:
             yield AttemptManager(self)
 
 
-class RetryException(Exception):
```

Comment:
I think this one is internal error IIRC. would need to double check.

Reply:
I see. @cdkrot could you confirm if `RetryException` and `RetriesExceeded` are internal error or not? If they are internal error, I think we can just close this PR.

Reply:
`RetryException` is an internal error but I think `RetriesExceeded` is not.

Reply:
Okay, I did a quick check. Let's only do it for `RetriesExceeded`

Reply:
Thanks for confirming! Will update the PR.